### DISCUSSION TITLE
Display Glucose notifications on Android Auto

### DISCUSSION THREAD

**Discussion by FranzMo** [NightscoutFoundation/xDrip#4087] [General]
It would be great if the xdrip App too ocmpatible with androidauto and the blood glucose value in the car could display.
https://www.android.com/auto/

**Comment by doc77** [comment]
You can currently have the Bg announced through the car speakers every 5 minutes with a simple Tasker profile.

I asked @jamorham about Android Auto compatibility a few weeks ago so it would display Bg and he declined because he thought it would be too distracting.

**Comment by jpsfs** [comment]
Hi!

To @FranzMo's point, and perhaps disagreeing with @jamorham, right now what happens is that I take the phone and look into it to check my BG, while I could get the same info while looking to the main screen of my car.

So, although I understand the rational, in fact, not having the feature there is more distracting than having it, just because if I truly want to know my BG I will know it, independently of having it on Android Auto or not.

I can volunteer to implement the feature if @jamorham gives is 👍 to it!

**Comment by jstevensog** [comment]
Hi All, and especially Jose,
I would also like Android Auto to display xDrip+ information, and have also
been thinking of implementing this.
Cheers

On Fri, Jan 18, 2019 at 1:27 AM José Pedro Silva <notifications@github.com>
wrote:

> Hi!
>
> To @FranzMo <https://github.com/FranzMo>'s point, and perhaps disagreeing
> with @jamorham <https://github.com/jamorham>, right now what happens is
> that I take the phone and look into it to check my BG, while I could get
> the same info while looking to the main screen of my car.
>
> So, although I understand the rational, in fact, not having the feature
> there is more distracting than having it, just because if I truly want to
> know my BG I will know it, independently of having it on Android Auto or
> not.
>
> I can volunteer to implement the feature if @jamorham
> <https://github.com/jamorham> gives is 👍 to it!
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/NightscoutFoundation/xDrip/issues/520#issuecomment-455190073>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIQs84KwhueePzoWeBhQPF73WyGlYeMuks5vEIhKgaJpZM4VcDWF>
> .
>


-- 
John Stevens
"You are how you live, not what you have."

**Comment by gregorybel** [comment]
for your information, I added android auto in aaps, see Dev branch. you see BG and Loop Info.

**Comment by lagerassassin** [comment]
Can I bump this, as @gregorybel has mentioned above, this seems possible on the face of it.

**Comment by lkiversen** [comment]
> for your information, I added android auto in aaps, see Dev branch. you see BG and Loop Info.

Hi. Where to find a possibility to see My looping data on android auto?

**Comment by gregorybel** [comment]
please have a look here: https://androidaps.readthedocs.io/en/latest/EN/Usage/Android-auto.html

**Comment by Navid200** [comment]
This sounds like a feature request.  if anyone can implement it, please do.  It would be wonderful if it was possible to snooze an alert while driving without having to use the phone.  So, I would consider that part of it an improvement. 

But, this is not really an issue.  I am going to convert it to a discussion.
